service/create/service.go

```diff
@@ -1281,7 +1248,8 @@ func (s *Service) onDelete(obj interface{}) {
 	s.logger.Log("info", fmt.Sprintf("cluster '%s' deleted", cluster.Name))
 }
 
-func (s *Service) onUpdate(oldObj, newObj interface{}) {
+// TODO we need to support this in operatorkit.
+func (s *Service) updateFunc(oldObj, newObj interface{}) {
```

Comment:
Yes we need update support for scaling and rolling workers. Once it's added we can use the reconciler her. Although most of the resources are in AWS not K8S.